### PR TITLE
Merlin 2.0.0 ide

### PR DESCRIPTION
**Pull request by SanjulaGanepola**

### CODE REVIEW

File: docs/guides/ide/editcl.md [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
-  After right clicking on a specific variable, users can select **Go to Definition** to go to the definition of the variable.  
+  Click a field or a variable in the Outline view to position the editor to its definition and highlight it.  
+  
+  ![CL Outline View Click](../../images/guides/idecloutlineview-2.png ':size=750')  
 
-- Content assist - code and variable completion  
-  Content assist can be triggered in editor window by typing **⌃/CTRL+SPACE**.  
+  Outline view supports filtering the items by name. Click on an item in the outline view and press `Ctrl+F` (`Cmd+F` on macOS). A popup will will appear at the top right corner of the outline view where you can now type the string that you would like to filter items by. There are also buttons next to the search box to `Filter` items or enable `Fuzzy Match`.  
 
-- Source error reporting - errors are shown beside the text in the editor and **Problems** view
```
Comment (SanjulaGanepola) [inline docs/guides/ide/editcl.md:16]: @eric-simpson Source error reporting for CL was previously in the docs, but it didn't appear to be a feature when I tried it out so I removed it from the docs. The CL extension [README](https://github.com/IBM/vscode-clle/blob/main/readme.md) also doesn't mention this as a feature. Was this added by mistake before or is this actually an implemented feature?

File: docs/guides/ide/startide.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Steps for the Merlin administrator:
```
Comment (SanjulaGanepola) [inline docs/guides/ide/startide.md:3]: @eric-simpson Above this line, the docs link to the [Merlin Overview](https://www.ibm.com/support/pages/node/6574839) support page which also looks to be outdated at least in terms of images. Do we need to update this as well?

File: docs/guides/ide/overview.md [hunk 3]
```diff
@@ -1,54 +1,106 @@
...
+
+Settings can be used to manage extension settings, keyboard shortcuts, themes (colors, file icons, and product icosn), and much more. To change any of these settings, click the **Manage** gear icon at the bottom left and select the appriopriate option. For information on settings, go to the [Settings](./guides/ide/settings.md) page.
+
+![Settings](../../images/guides/ideWorkspaceSettings.png ':size=350')
 
+To return to the dashboard, click the white **IBM i Developer Workspaces Web** button in the bottom left status bar and use the **IBM i Developer Workspaces: Open Dashboard** command.
 
+![Return to Dashboard](../../images/guides/ideWorkspaceReturn.png ':size=800')
 
 For information on using ARCAD tools, see [How to use ARCAD integration with IBM i Modernization Engine for Lifecycle Integration](https://supportcontent.ibm.com/support/pages/how-use-arcad-integration-ibm-i-modernization-engine-lifecycle-integration-merlin)
```
Comment (SanjulaGanepola) [inline docs/guides/ide/overview.md:108]: @eric-simpson Almost all the IDE pages have this same exact text at the end of each page. Can I remove this and just add a page that briefly explains the ARCAD tools and links to their docs site? Also the content on this support page is outdated. Do we need to update it or can it be removed?

File: docs/guides/ide/extensions.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Extensions
+
+## Installed Extensions
+
+The **Extensions** view is where all VS Code extensions will be managed including several extensions which are automatically installed when creating a new workspace. These extensions add languages, debuggers, and tools to support your development workflow.
```
Comment (eric-simpson) [inline docs/guides/ide/extensions.md]: The **Extensions** view is where all VS Code extensions will be managed including several extensions which are automatically installed when creating a new workspace. These extensions add languages, debuggers, and tools to support your IBM i development workflow.  Additional extensions can be installed that supplement the capabilities in the development environment.

File: docs/guides/ide/extensions.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+![Extensions View](../../images/guides/extensionsView.png ':size=500')
+
+The following extensions will be automatically installed:
```
Comment (eric-simpson) [inline docs/guides/ide/extensions.md]: no need to list the extensions provided automatically

File: docs/guides/ide/extensions.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+* **RPGLE** - Language tools and linter for RPGLE.
+* **CL** - Language tools for CLLE.
+* **IBM i Languages** - Provides syntax highlighting for IBM i languages such as RPG, CL, DDS, MI, and RPGLE fixed/free.
+* **vscode-sourceorbit** - Object dependency management.
+* **Code for IBM i Walkthroughs** - Walkthroughs relevant to **Code for IBM i**.
+* **ARCAD-Elias** - ARCAD tools (**ARCAD-Skipper**, **ARCAD-Observer**, and **ARCAD-Transformer**).
+
+## Install an Extension
+
+The [Visual Studio Code Extension Marketplace](https://marketplace.visualstudio.com/VSCode) is where extension authors publish extensions that extend the features in VS Code. You may browse for any of these extensions directly in the **Extensions** view. To narrow down your search, use filters such as `@popular` or `@category:"programming languages"`.
```
Comment (eric-simpson) [inline docs/guides/ide/extensions.md]: remove 
To narrow down your search, use filters such as `@popular` or `@category:"programming languages"`.

need to look for text about IBM disclaimer.  IBM does not support extensions from the marketplace

File: docs/guides/ide/extensions.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+![Browse for Extensions](../../images/guides/extensionsMarketplace.png ':size=600')
+
+After you have found an extension you would like to install, select it to see the Extension details page. Here you can find the extension ID, publisher, number of downloads, ratings, general overview of the extension. After selecting the **Install** button, VS Code will download and install the extension from the Marketplace. Once the installation is complete, the **Install** button will be replaced by a **Disable** and **Uninstall** button.
+
+![Install Extension](../../images/guides/extensionsInstall.png ':size=750')
+
+Extensions may also be installed from the [Open VSX Registry](https://open-vsx.org/), by downloading the `VSIX` file of an extension you would like to install and then using the **Install from VSIX...** action.
```
Comment (eric-simpson) [inline docs/guides/ide/extensions.md]: Extensions may also be dowloaded from the [Open VSX Registry](https://open-vsx.org/) and then installed using the **Install from VSIX...** action.

File: docs/guides/ide/editrpg.md [hunk 8]
```diff
@@ -1,59 +1,78 @@
...
-   ![RPG hover](../../images/guides/iderpghover.png)
 
-- references - show particular code element is referenced throughout the codebase  
-  After right clicking on a specific variable, users can select either **Go to References** or **Peek References** to show all references embedded inline. You can navigate between different references in the peeked editor and make quick edits right there.
-   ![RPG Go to References](../../images/guides/iderpggotoreferences.png)
+  ![RPG hover](../../images/guides/iderpghover.png ':size=800')  
 
-  Users can also select **Find all references**, but will manually need to open the references view from the left sidebar, to reveal the references.  
-   ![RPG Find all References](../../images/guides/iderpgfindallreferences.png)
+- References - Show where a particular code element is referenced throughout the codebase.
```
Comment (eric-simpson) [inline docs/guides/ide/editrpg.md]: what is codebase?  the workspace?

File: docs/guides/ide/edit.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Editing
   
-The (Integrated Development Environment)IDE is based on Eclipse Che/Theia and it has a similar user experiences as Visual Studio (VS) Code.
+The (Integrated Development Environment) IDE is Visual Studio Code running in Devspaces and it comes equiped with several IBM i related extensions that provide editing support.
```
Comment (eric-simpson) [inline docs/guides/ide/edit.md]: running in Devspaces
should be
running in Red Hat OpenShift Dev Spaces

change 'comes equiped with several' to 'includes'

File: docs/guides/ide/startide.md (hunk 2 above)
Comment (eric-simpson) [reply docs/guides/ide/startide.md:3]: the link to the merlin overview support page should be removed

File: docs/guides/ide/overview.md (hunk 3 above)
Comment (eric-simpson) [reply docs/guides/ide/overview.md:108]: this will be updated later across all the pages

File: docs/guides/ide/editcl.md (hunk 1 above)
Comment (eric-simpson) [reply docs/guides/ide/editcl.md:16]: matching what is in the cl extension readme is correct

File: docs/guides/ide/extensions.md (hunk 6 above)
Comment (SanjulaGanepola) [reply docs/guides/ide/extensions.md]: > need to look for text about IBM disclaimer. IBM does not support extensions from the marketplace

Does this text ("IBM does not support extensions from the marketplace") need to be included here?

File: docs/guides/ide/editrpg.md (hunk 8 above)
Comment (SanjulaGanepola) [reply docs/guides/ide/editrpg.md]: Updated to "workspace"

File: docs/guides/ide/extensions.md (hunk 6 above)
Comment (eric-simpson) [reply docs/guides/ide/extensions.md]: yes, as a Note

File: docs/guides/ide/editcl.md [hunk 10]
```diff
@@ -1,20 +1,39 @@
 # Editing CL
 
-Apart from those common language features mentioned in [Edit](edit.md), CL source editing also has these capabilities:
+Apart from those common language features mentioned in [Edit](edit.md), CL source editing also has the following capabilities:
```
Comment (william-xiang) [inline docs/guides/ide/editcl.md]: Seems like this edit.md file cannot be referenced. Need to change the link to be `./guides/ide/edit.md`. The other places may also have the same issue.

**Comment by william-xiang** [comment]
@SanjulaGanepola For start workspace part, could you add the option `Import from Git`? Also for ssh format repo url, need to create ssh key in credential first.

**Review by jonnyz32 — APPROVED** [review]

File: docs/guides/ide/editcl.md (hunk 10 above)
Comment (SanjulaGanepola) [reply docs/guides/ide/editcl.md]: Thanks for catching that

**Comment by SanjulaGanepola** [comment]
> @SanjulaGanepola For start workspace part, could you add the option Import from Git? Also for ssh format repo url, need to create ssh key in credential first.

@william-xiang Added it -> a14662d

File: docs/guides/cicd/Manage_Logs.md [hunk 11]
```diff
@@ -1,3 +1,16 @@
-IBM i CI/CD supports configuring log levels and viewing GUI logs.
-* [Configure Log Level](Configure_Log_Level.md)
-* [View Logs](View_Logs.md) 
\ No newline at end of file
+# Manage Logs
+
+## Configure Log Level
+IBM i CI/CD GUI supports configuring the log level. The default log level is INFO, but that sets it to a different level is supported.
```
Comment (eric-simpson) [inline docs/guides/cicd/Manage_Logs.md]: The default log level is INFO, but that sets it to a different level is supported.  
should be
 The default log level is INFO.  The log level can be set to a different level.

File: docs/guides/ide/developerbuild.md [hunk 12]
```diff
@@ -4,87 +4,150 @@ IBM i Developer allows the user to run the project-based developer builds. IBM i
...
+If the action above is selected, all successful commands are filtered out so that you can focus on the failing ones.  This can be very useful when lots of objects are being built.
+
+![Show All Jobs](../../images/guides/devBuildShowAllJobs.png ':size=800')
+
+To help zero in on the problem, there is also an action to filter the messages, so that you can set the minimum severity of messages you would like to see. In the example below, the message severity was set to 10 so that all of the information messages were filtered out.
+
+![Filter Message Severity](../../images/guides/devBuildFilterMessageSeverity.png ':size=750')
+
+### Object Level
+The children of a job log are all the ILE objects being built.  The description is the source in the IFS that is being compiled into that object.
```
Comment (eric-simpson) [inline docs/guides/ide/developerbuild.md]: remove 'ILE" and 'being'

**Review by william-xiang — APPROVED** [review]
Approved. Thank you.

File: docs/guides/platform/ManageInventory.md [hunk 13]
```diff
@@ -1,30 +1,24 @@
...
 
-<img src="../../images/guides/Pvcinventory.png" width = "400" height = "300" alt="Add power vc inventory" align=center />  
+  <img src="../../images/guides/Addinventory.png" width = "400" height = "250" alt="Add Inventory" align=center />    
 
-Specify the Hostname and Port, the Test TLS button will be enabled, and it will catch the Power VC CA.  In the detailed certificate windows, click the Accept button.  Then this CA be added to Merlin trust store.
+  For the security connections, Merlin supports the `Test TLS` function which allows customers to accept the remote CA such as for a Power VC Inventory. After specifying the `Hostname` and `Port`, the `Test TLS` button will be enabled and it will catch the Power VC CA. In the detailed certificate windows, click the `Accept` button. Then this CA be added to the Merlin trust store.
```
Comment (eric-simpson) [inline docs/guides/platform/ManageInventory.md]: 'For the security connections', does this mean 'secure connection'?

Comment (SanjulaGanepola) [reply docs/guides/platform/ManageInventory.md]: Yes, that was a typo. Fixed it

**Comment by eric-simpson** [comment]
@SanjulaGanepola can you resolve the merge conflicts?

**Comment by SanjulaGanepola** [comment]
@eric-simpson Conflicts are resolved

**Pull request merged**